tools/assets/stylesheets/avaiga.css

```diff
@@ -1,25 +1,40 @@
+@import 'partials/_grid.css';
+
 :root {
   --md-primary-fg-color: #293183;
   --md-primary-fg-color--light: #f6921b;
   --md-primary-fg-color--dark: #ec693c;
+  --md-paper-bg-color: #fff;
+  --transition-duration-base: 400ms;
+  --tooltip-show-delay: 400ms;
+}
+[data-md-color-scheme="slate"] {
+  --md-hue: 240;
+  --md-paper-bg-color: #140a1e;
 }
+
 .md-header__button.md-logo img {
   width: 48px;
   height: 48px;
 }
 
-[data-md-color-scheme="slate"] {
-  --md-hue: 240;
+.md-content{
+  overflow: initial;
 }
 
-a.ext-link:after {
+/* Display the right icon for the right theme */
+.icon-light{ display: none; }
+[data-md-color-scheme="slate"] .icon-dark{ display: none; }
+[data-md-color-scheme="slate"] .icon-light{ display: inline-block; }
+
+a.ext-link::after {
   display: inline-block;
   content: url("../images/ext-link.svg");
   width: 0.6em;
   height: 0.6em;
   margin: 0 3px 0 3px;
 }
-[data-md-color-scheme="slate"] a.ext-link:after {
+[data-md-color-scheme="slate"] a.ext-link::after {
```

Comment:
Just so that I learn something... why two colons?

Reply:
`::before` and `::after` are pseudo-elements (like `::placeholder`, for instance) that refer to a virtual CSS element whereas things like `:hover` or `:focus` are pseudo-classes that refer to an element's state.
`:after` may be well interpreted valid CSS, but the actual good practice for this is to make the distinction between pseudo-elements (two columns) and pseudo-classes (valid only with one column) in the code.

Reply:
`::before` and `::after` are pseudo-elements (like `::placeholder`, for instance) that refer to a virtual CSS element whereas things like `:hover` or `:focus` are pseudo-classes that refer to an element's state.
`:after` may be well interpreted valid CSS, but the actual good practice for this is to make the distinction between pseudo-elements (two colons) and pseudo-classes (valid only with one colon) in the code.